examples/lock-app/nrfconnect/README.md

```diff
@@ -1,53 +1,101 @@
-# CHIP nRF Connect nRF52840 Lock Example Application
+# CHIP nRF Connect Lock Example Application
 
-An example application showing the use
-[CHIP](https://github.com/project-chip/connectedhomeip) on the Nordic nRF52840.
+The lock example provides demonstration of application running on the connected
```

Comment:
I think we should mention Nordic in the text. How about something like:
"The nRF Connect lock example provides a working demonstration of a
connected door lock device targeting Nordic Semiconductor development kit boards.
State of the simulated lock bolt can be controlled remotely over a CHIP network or using a physical button."
It's just a suggestion, but I think it should be clear in the paragraph which board manufacturer the example is for.